workers/unity/Packages/io.improbable.gdk.testutils/TestSettings/il2cpp-dotnet-4.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Also same question, does these files belong in this package? It feels like they belong to the `playground` project itself

Reply:
Could argue they're generic enough to be in a module, but I don't really mind where these files go

Reply:
Moved them